<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/images/globe-outline.svg</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="showToolbarAction"/>
    <addaction name="showStatusBarAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="processingMenu">
    <property name="title">
     <string>Processin&amp;g</string>
    </property>
    <addaction name="runAction"/>
    <addaction name="separator"/>
    <addaction name="protectionAction"/>
   </widget>
   <widget class="QMenu" name="languageMenu">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="processingMenu"/>
   <addaction name="languageMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="separator"/>
   <addaction name="runAction"/>
   <addaction name="protectionAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
  </widget>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/exit-outline.svg</normaloff>:/images/images/exit-outline.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/information-circle-outline.svg</normaloff>:/images/images/information-circle-outline.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>Show program information</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="showToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;toolbar</string>
   </property>
   <property name="toolTip">
    <string>Show or hide the toolbar</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/folder-open-outline.svg</normaloff>:/images/images/folder-open-outline.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file to work with</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="runAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/arrow-forward-circle-outline.svg</normaloff>:/images/images/arrow-forward-circle-outline.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run processing on current file</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="showStatusBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;statusbar</string>
   </property>
   <property name="toolTip">
    <string>Show status bar at the bottom of window</string>
   </property>
  </action>
  <action name="protectionAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/shield-outline.svg</normaloff>:/images/images/shield-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Enable &amp;protection</string>
   </property>
   <property name="toolTip">
    <string>Enable protection when running</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>showToolbarAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showStatusBarAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
